TRANSLATION_GUIDE.md

```diff
@@ -0,0 +1,32 @@
+ Translation Guide
+ ======
+
+This project uses [ICU message format](https://formatjs.io/docs/core-concepts/icu-syntax/).
+
+ ## Formatted arguments
+
+[Formatted arguments](https://formatjs.io/docs/core-concepts/icu-syntax/#formatted-argument) are the preferred way of formatting. Custom formatting functions are allowed if the output is not part of the message (i.e. non translated value embeded directly in the markup) or if required formatting cannot be achieved using ICU format.
+ 
+ ### Number skeleton
+
+Common number formatting options (in concise syntax) and their mapping to Intl.NumberFormat properties (used for custom formatting functions):
+1. `minimumIntegerDigits: 1` -> `0` ([integer-width](https://unicode-org.github.io/icu/userguide/format_parse/numbers/skeletons.html#integer-width))
+2. `minimumFractionDigits: 0`, `maximumFractionDigits: 0` -> `.` (equivalent to [precision-integer](https://unicode-org.github.io/icu/userguide/format_parse/numbers/skeletons.html#precision) )
+3. `minimumFractionDigits: 1`, `maximumFractionDigits: 1` -> `.0` ([fraction-precision](https://unicode-org.github.io/icu/userguide/format_parse/numbers/skeletons.html#fraction-precision))
+4. `useGrouping: false` -> `,_` ([group-off](https://unicode-org.github.io/icu/userguide/format_parse/numbers/skeletons.html#grouping))
+5. `style: 'percent'` -> `%` ([percent](https://unicode-org.github.io/icu/userguide/format_parse/numbers/skeletons.html#unit) unit)
+
+Example:
```

Comment:
Add an anticipated result in addition to what the format will look like.  Just to connect the doc, example and output together...for example:

"Given the following message:
```js
{
  demoFormat: {
    id: 'demoFormat',
    defaultMessage: 'This shows {value, number, ......} percent as the  formatted output',
  }
}
```

when we call `msg.demoFormat({ value: 0.125 })` the output will be:
```
  This shows 13% as the formatted output.
```
"

Reply:
You didn't add an output for the example, as Scott suggested. I think it worth adding....

Reply:
@rszwajko are you planning to add an example, not something fancy, just show that the msg example mentioned below for  `dashboardUtilizationCardAvailableOfPercent` can be translated to:....if `percent` is set to....

Reply:
Sure. Added the output.